Tests/test_SeqIO_features.py

```diff
@@ -80,16 +87,22 @@ def compare_record(old, new, expect_minor_diffs=False):
             continue
         if key == "molecule_type":
             # EMBL allows e.g. "genomics DNA" where GenBank is limited.
-            common_words = set(old.annotations[key].split()).intersection(new.annotations[key].split())
+            common_words = set(old.annotations[key].split()).intersection(
+                new.annotations[key].split()
+            )
```

Comment:
This doesn't look so nice... the one line version must be right on the 88 limit. Perhaps two lines?

```python
common_words = set(old.annotations[key].split())
common_words = common_words.intersection(new.annotations[key].split())
```

Reply:
amended, Thank you.